packages/core/redux/multisigWallet/actions.js

```diff
@@ -232,11 +232,12 @@ export const initMultisigWalletManager = () => async (dispatch, getState) => {
     return
   }
   dispatch({ type: MULTISIG_INIT, isInited: true })
+  const { account } = getState().get(DUCK_SESSION)
```

Comment:
Please use appropriate model selector instead of 'direct' access. Find it at `ChronoMint/packages/core/redux/session/selectors/models.js` (named `getAccount`)